client/k8sextclient/k8sextclient.go

```diff
@@ -44,20 +44,6 @@ type Config struct {
 	TLS       TLSClientConfig
 }
 
-// DefaultConfig provides a default configuration to create a new Kubernetes
-// Clientset by best effort.
-func DefaultConfig() Config {
-	return Config{
-		// Dependencies.
-		Logger: nil,
-
-		// Settings.
-		Address:   "",
-		InCluster: true,
```

Comment:
I think in our operators we should be fine because we set this usually in the config maps. We cannot really argue if we need to default a boolean to something else. This is for instance why the AWS SDK only deals with pointers in the input settings.